src/components/appShell/icons/DashboardIcon.tsx

```diff
@@ -0,0 +1,5 @@
+import { ReactComponent as DashboardSvg } from '../../../assets/icons/Dashboard.svg';
+
+export const DashboardIcon = ({ w, h }: { w: string; h: string }) => {
```

Comment:
we need some kind of a library to handle these stuff (sizing and coloring) instead of doing it manually
mantine does't have a  `Icon` component but does have a [ThemeIcon](https://mantine.dev/core/theme-icon/) which we need to customize it since it has some default props (like background and etc) or we can use something else

any ideas? @ASafaeirad @mojtabaesl

Reply:
I agree with ThemeIcon

Reply:
@erfan-goodarzi if you mean the background and the borders (the default styles)
we can change mantine components styles globally [like this](https://mantine.dev/theming/default-props/)